### PR TITLE
Update Hong Kong Page

### PR DESCRIPTION
**Pull request by anscg**

### CODE REVIEW

**Comment by lachlanjc** [comment]
#### 📝 Changed routes:


| [**/hongkong**](https://counterspell-f846wn208.hackclub.dev/hongkong) |
|:-:|
|<a href="https://counterspell-f846wn208.hackclub.dev/hongkong">Screenshot of /hongkong</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/K0BW0gEmIZHDGAvdYnm4Z2HkMLIi9UtPAQTbpVtPYgvTPdS0vz1T9EYhCDcb7rNocKD2qRsrUZUzvb6TZHMbvQ.png">(view full size)</a> |







Commit 5068c0dc51aff93a8d256a0afd7bf1302d1cf20b (https://counterspell-f846wn208.hackclub.dev).

File: pages/hongkong.js [hunk 1]
```diff
@@ -45,6 +45,13 @@ export default function HongKong() {
...
+            posthog.init('phc_6ZwHOIl8Q3SQVW3wyJtEeiBAEyy7wO90wE5jJEMftk1',{api_host:'https://data.anscg.workers.dev', person_profiles: 'always' 
+            })
+          `}
+        </script>
```
Comment (samdev-7) [inline pages/hongkong.js:54]: What's this for?

Comment (anscg) [reply pages/hongkong.js:54]: It's posthog

Comment (anscg) [reply pages/hongkong.js:54]: https://posthog.com/docs/getting-started/install?tab=snippet

**Review by samdev-7 — APPROVED** [review]

**Comment by samdev-7** [comment]
Note: deployment has been fixed in prod. Typo on my end.

**Pull request merged**